Client/electrotech/src/pages/home/Home.js

```
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import LayoutApp from "../../components/Layout";
+import Grid from "@material-ui/core/Grid";
+import Product from "../../components/Product";
+import { useDispatch } from "react-redux";
+import { Button } from "@material-ui/core";
+
+const Home = () => {
+  const dispatch = useDispatch();
+  const [productData, setProductData] = useState([]);
+  const [categories, setCategories] = useState([
+    "laptops",
+    "smart phones",
+    "home applicants",
+  ]);
+  const [selectedCategory, setSelectedCategory] = useState("laptops");
+
+  useEffect(() => {
+    const getAllProducts = async () => {
+      try {
+        dispatch({
+          type: "SHOW_LOADING",
+        });
+
+        // const { cate } = await axios.get("http://localhost:3030/categories");
+        // setCategories(cate);
+        // console.log(cate);
+
+        const { data } = await axios.get("http://localhost:3030/products");
+        setProductData(data);
+        // console.log(data);
+
+        dispatch({
+          type: "HIDE_LOADING",
+        });
+      } catch (e) {
+        console.log(e);
```

no consoles and no empty catches